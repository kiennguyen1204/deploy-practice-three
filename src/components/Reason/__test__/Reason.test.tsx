import { render, screen } from '@testing-library/react';

// components
import WhatMake from 'components/Reason';

describe('WhatMake', () => {
  it('renders the component with the correct title', () => {
    const { asFragment } = render(<WhatMake />);
    const title = screen.getByTestId('what-make-title');
    expect(title).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
