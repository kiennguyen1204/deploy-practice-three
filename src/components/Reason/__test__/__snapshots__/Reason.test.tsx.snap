// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WhatMake renders the component with the correct title 1`] = `
<DocumentFragment>
  <article
    class="what-make-box-container"
  >
    <section
      class="wrapper-element"
    >
      <h3
        class="title"
        data-testid="what-make-title"
      >
        WHAT MAKES US THE 
        <span
          class="num-1"
        >
          #1
        </span>
         ONLINE MARIJUANA DISPENSARY IN CANADA?
      </h3>
      <p
        class="description"
      >
        When it comes to what makes us the foremost online marijuana dispensary in Canada, we could wax lyrical about our positive qualities. Instead, to make this information clearer, we’ve highlighted the six prioritized features that we feel makes us a cut above the rest.
      </p>
      <div
        class="content-box-container"
      >
        <section
          class="item-box"
        >
          <div
            class="icon"
          >
            <img
              alt="Customer Service"
              src="IMAGE_MOCK"
            />
          </div>
          <p
            class="title-box"
          >
            CUSTOMER SERVICE
          </p>
          <p
            class="description"
          >
            Whether it is answering any questions you have before making a purchase, helping out with the buying process itself or taking your feedback under consideration, we are proud to provide high quality customer service that makes you – the customer – the most important person in the transaction.
          </p>
        </section>
        <section
          class="item-box"
        >
          <div
            class="icon"
          >
            <img
              alt="SECURITY"
              src="IMAGE_MOCK"
            />
          </div>
          <p
            class="title-box"
          >
            SECURITY
          </p>
          <p
            class="description"
          >
            When it comes to security, we only keep what details are necessary for you to have an account with us and make an order. When it comes to shipping your mail order marijuana out, we use only tamper-proof and discrete packaging so then what you’ve purchased is your business only.
          </p>
        </section>
        <section
          class="item-box"
        >
          <div
            class="icon"
          >
            <img
              alt="DELIVERY INSURANCE"
              src="IMAGE_MOCK"
            />
          </div>
          <p
            class="title-box"
          >
            DELIVERY INSURANCE
          </p>
          <p
            class="description"
          >
            If your mail order marijuana becomes lost, stolen, or damaged during transit, we will send you a replacement completely free of charge. Free Canada Post Xpress shipping on all orders over $120
          </p>
        </section>
        <section
          class="item-box"
        >
          <div
            class="icon"
          >
            <img
              alt="BEST VALUE"
              src="IMAGE_MOCK"
            />
          </div>
          <p
            class="title-box"
          >
            BEST VALUE
          </p>
          <p
            class="description"
          >
            We are continually adjusting what we supply and our prices to ensure that we maintain an optimal balance of affordable and quality for our products. We invest in the best quality strains that we can find and are always on the lookout for new, affordable and high quality weed products.
          </p>
        </section>
        <section
          class="item-box"
        >
          <div
            class="icon"
          >
            <img
              alt="TRUST"
              src="IMAGE_MOCK"
            />
          </div>
          <p
            class="title-box"
          >
            TRUST
          </p>
          <p
            class="description"
          >
            With over 15 years in the weed business, you can rest assured that you will be taken care of. You can guarantee that we have your best interests in mind. Feel free to check out our reviews.
          </p>
        </section>
        <section
          class="item-box"
        >
          <div
            class="icon"
          >
            <img
              alt="HIGHEST QUALITY"
              src="IMAGE_MOCK"
            />
          </div>
          <p
            class="title-box"
          >
            HIGHEST QUALITY
          </p>
          <p
            class="description"
          >
            All of our cannabis products are tested to ensure that they are the highest quality possible. We work with expert suppliers and are always revising what makes a quality cannabis product to ensure that we have only the best available.
          </p>
        </section>
      </div>
    </section>
  </article>
</DocumentFragment>
`;
